jpo-ode-svcs/src/main/java/us/dot/its/jpo/ode/traveler/TimDeleteController.java

```diff
@@ -43,6 +53,14 @@
       }
 
       RSU queryTarget = (RSU) JsonUtils.fromJson(jsonString, RSU.class);
+      
+      if (queryTarget.getRsuUsername() == null || queryTarget.getRsuUsername().isEmpty()) {
```

Comment:
SHould this be `curRsu.getRsuUsername() != null && !curRsu.getRsuUsername().isEmpty()`